editions/de-AT-server/tiddlywiki.info

```diff
@@ -2,7 +2,8 @@
 	"description": "Server configuration of the German (Österreich) edition",
 	"plugins": [
 		"tiddlywiki/tiddlyweb",
-		"tiddlywiki/filesystem"
+		"tiddlywiki/filesystem",
+		"tiddlywiki/highlight"
```

Comment:
I think '$:/config/Plugins/Disabled/$:/plugins/tiddlywiki/highlight' is set 'no' instead of Highlight plugin is removed.